gdrive_ros/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(gdrive_ros)
+
+find_package(catkin REQUIRED COMPONENTS
+    std_msgs
+    message_generation
+)
+
+add_service_files(
+  FILES
+  Upload.srv
+  MultipleUpload.srv
+)
+
+generate_messages(
+  DEPENDENCIES
+  std_msgs
+)
+
+catkin_package(
+  CATKIN_DEPENDS
+  message_runtime
+  std_msgs
+)
```

Comment:
Please install `node_scripts` directory to `CATKIN_PACKAGE_BIN_DESTINATION` or `CATKIN_PACKAGE_SHARE_DESTINATION`, and `launch` directory to `CATKIN_PACKAGE_SHARE_DESTINATION`